### ISSUE TITLE
The "View On Instance" button's tooltip should say "View On Instance" instead of simply "Link"

### ISSUE THREAD

**Issue by K4LCIFER** [LemmyNet/lemmy-ui#2024]
### Requirements

- [X] This is a feature request and not a bug report. Otherwise, please create a new [bug report](https://github.com/LemmyNet/lemmy-ui/issues/new?assignees=&labels=bug%2Ctriage&projects=&template=BUG_REPORT.yml) instead.
- [X] Please [check](https://github.com/LemmyNet/lemmy-ui/issues) to see if this request (or a similar one) already exists.
- [X] It's a single feature. Please don't request multiple features in one issue.

### Describe the feature you'd like

As shown in the following picture, the "View On Instance" button currently just says "Link":

![image](https://github.com/LemmyNet/lemmy-ui/assets/7829231/5d98bb64-4d38-42db-af9b-bfc4e4f1642e)

I propose that it, instead, says "View On Instance", to be clearer in its purpose to the user.

**Comment by beedell-roke** [comment]
@dessalines, isn't this an accessibility concern as much as a usability one? I don't think closing this is wise. I know it would impact downstream instances, like KDE's, if the user can't navigate the website with a screen reader.

**Comment by dessalines** [comment]
The a title of that is actually fedilink, not link, which is what screen readers would see. 

If you want something more specific, lmk and I'll re-open and assign it to you.

**Comment by beedell-roke** [comment]
> https://github.com/LemmyNet/lemmy-ui/issues/2024#issuecomment-3136940225

@dessalines, insofar as screen readers can differentiate it, that's <em>sufficient</em> for me. Thanks.

**Comment by hackerncoder** [comment]
> The a title of that is actually fedilink

Not anymore? I started using orca to test out things, and it says "link link" for the context link and "link link" for the fediverse context link

**Comment by Die4Ever** [comment]
> > The a title of that is actually fedilink
> 
> Not anymore? I started using orca to test out things, and it says "link link" for the context link and "link link" for the fediverse context link

I'm seeing `title="link"` on 0.19.3, which I guess is where that's coming from, probably both should be fedilink

```
<a class="btn btn-sm btn-link btn-animate text-muted py-0" title="link" href="https://lemmy.wtf/post/33163741"><svg class="icon icon-inline"><use xlink:href="/static/39a7b2dd/assets/symbols.svg#icon-fedilink"></use><div class="visually-hidden"><title>fedilink</title></div></svg></a>
```

**Comment by K4LCIFER** [comment]
> > The a title of that is actually fedilink
> 
> Not anymore? I started using orca to test out things, and it says "link link" for the context link and "link link" for the fediverse context link

I'm still  seeing "link" in 0.19.13:

<img width="679" height="160" alt="Image" src="https://github.com/user-attachments/assets/84127c87-6d52-4ba6-8b72-1244f44963b3" />

**Comment by K4LCIFER** [comment]
> The a title of that is actually fedilink, not link, which is what screen readers would see.

Why doesn't it display "fedilink" then instead of "link"?

**Comment by dessalines** [comment]
Its fixed in main. Feel free to do a PR if you want it backported.
